### ISSUE TITLE
de.marmaro.krt.ffupdater.MainActivity$UpdateCheckException at MainActivity.krt: 204

### ISSUE THREAD

**Issue by SlidingHorn** [Tobi823/ffupdater#52]
This crash came up on my automatic check for updates and continues if I try to manually check for updates:

```
java.lang.NullPointerException
	at de.marmaro.krt.ffupdater.app.impl.fetch.mozillaci.MozillaCiLogConsumer.updateCheck(MozillaCiLogConsumer.kt:28)
	at de.marmaro.krt.ffupdater.app.impl.FirefoxRelease.updateCheckWithoutCaching(FirefoxRelease.kt:39)
	at de.marmaro.krt.ffupdater.app.BaseAppDetail$updateCheck$$inlined$withLock$lambda$1.invokeSuspend(BaseAppDetail.kt:63)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
```

**Comment by Tobi823** [comment]
Thanks for the info - I've found the root cause for this bug

**Comment by Tobi823** [comment]
Fixed FFUpdater version 74.3.0 should be available soon on F-Droid (but can take couple days)

**Comment by SlidingHorn** [comment]
> Fixed FFUpdater version 74.3.0 should be available soon on F-Droid (but can take couple days)

Looks like a build started a little under 2 hours ago, and the new version was picked up and successfully built. Now we all just need to wait for the rest of the build to finish and for it to be signed.

Thanks for finding/fixing this so quickly!

**Comment by Tobi823** [comment]
Update is available on F-Droid
